libc/src/__support/RPC/rpc.h

```diff
@@ -67,18 +76,18 @@ template <bool Invert> struct Process {
   LIBC_INLINE ~Process() = default;
 
   uint32_t port_count = 0;
-  cpp::Atomic<uint32_t> *inbox = nullptr;
-  cpp::Atomic<uint32_t> *outbox = nullptr;
+  uint32_t *inbox = nullptr;
```

Comment:
Not sure which approach is better. With `std::atomic<>`, it’s clear that the variable is an atomic number. However, as a simple integer, its usage may not be safe if someone forgets to update it using atomic operations.

Reply:
It's internal to the class, so we don't need to worry about anything external. If someone in the future modifies this I'll hope that they see that every access to this is atomic and that there's atomic in the comments.